profile/isa_ro_crate_mapping.md

```diff
@@ -0,0 +1,157 @@
+|date|endTime||
+|previousProcess|-|redundant information|
+|nextProcess|-|redundant information|
+|comments|disambiguatingDescription||
+||
+|Protocol|[bioschemas.org/LabProtocol](https://bioschemas.org/LabProtocol)|
+|@id|@id||
+|name|name||
+|protocolType|intendedUse||
+|description|description||
+|uri|url||
+|-|comment|ARC-specific property|
+|version|version||
+|components|labEquipment|Components can be saved as `labEquipment`, `reagent`, or `computationalTool`, with `labEquipment` being the default.|
+|components|reagent|Components can be saved as `labEquipment`, `reagent`, or `computationalTool`, with `labEquipment` being the default.|
+|components|computationalTool|Components can be saved as `labEquipment`, `reagent`, or `computationalTool`, with `labEquipment` being the default.|
+|parameters|-|redundant information|
+||
+|Sample|[bioschemas.org/Sample](https://bioschemas.org/Sample)||
+|Source|[bioschemas.org/Sample](https://bioschemas.org/Sample)||
+|@id|@id||
+|name|name||
+|characteristics|additionalProperty|Characteristics and factor values can be saved as `additionalProperties`. What they represent needs to be annotated.|
+|factorValues|additionalProperty|Characteristics and factor values can be saved as `additionalProperties`. What they represent needs to be annotated.|
+|derivesFrom|-|redundant information|
+||
+|Data|'File' or 'MediaObject'|
+|@id|@id||
+|name|name||
+|comments|comment||
+|-|encodingFormat|ARC-specific property|
+|type|disambiguatingDescription||
+||
+|@type |[schema.org/Person](https://schema.org/Person)|
+|@id|@id||
+|firstName|givenName||
+|lastName|familyName||
+|email|email||
+|-|identifier|Can be used for identifiers stored in comment, e.g. an orcid.|
+|affiliation|affiliation|The affilitation strin in ISA is converted to an Oranization object.|
+|roles|jobTitle||
+|midInitials|additionalName||
+|address|address||
+|phone|telephone||
+|fax|faxNumber||
+|comments|disambiguatingDescription|Comment object in ISA has to be encoded as a string.|
+||
+|Publication|[schema.org/ScholarlyArticle](https://schema.org/ScholarlyArticle)|
+|@id|@id||
+|title|headline||
+|pubMedID|identifier|One or many identifiers for this article (DOI or PubMedID) can be encoded in identifier. Either as full URL or of type PropertyValue to indicate the kind of reference.|
+|doi|identifier|One or many identifiers for this article (DOI or PubMedID) can be encoded in identifier. Either as full URL or of type PropertyValue to indicate the kind of reference.|
+|authorList|author||
+|status|creativeWorkStatus|
+|comments|comment||
+||
+|Comment|Comment||
+|@id|@id||
+|name|name||
+|value|text||
+||
+|OntologyAnnotation|DefinedTerm||
+|annotationValue|name||
+|termSource|inDefinedTermSet||
+|termAccession|termCode||
+|comments|disambiguatingDescription|Comment object in ISA has to be encoded as a string.|
+||
+|FactorValue|[schema.org/PropertyValue](https://schema.org/PropertyValue)|See details for properties below.|
```

Comment:
When section below has a header, link to it?

Reply:
yeah, I think this would help.